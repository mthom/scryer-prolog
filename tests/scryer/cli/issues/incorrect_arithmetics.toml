args = [
    "-f",
    "--no-add-history",
    "-g", "main,halt",
    "incorrect_arithmetics.pl"
]
