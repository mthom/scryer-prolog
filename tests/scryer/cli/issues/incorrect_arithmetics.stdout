% Info: Selected 1432th out of 2136 found aritmetic relations that satisfy ttttt template:
	A is+sqrt(-e).
% Warning: singleton variables A at line 0 of chnytjl.pl
